modules/claims_api/spec/lib/claims_api/local_bgs_spec.rb

```diff
@@ -19,7 +19,7 @@
         # 2: connection_wsdl_get - duration of WSDL request cycle
         # 3: connection_post - how long does the post itself take for the request cycle
         # 4: parsed_response - how long to parse the response
-        expect(ClaimsApi::Logger).to receive(:log).exactly(4).times
+        expect(ClaimsApi::Logger).to receive(:log).exactly(3).times
```

Comment:
I think we should maintain the test coverage in this spec file over the legacy code path until that code path is actually gone. Relatedly, this spec file can be more explicit about exercising the different stages for our rollout.  Those stages correspond to the combinations of the two booleans `defined?` and `enabled?`.

But I don't see any Flipper logic included. I believe @FonzMP wanted a quick toggle back to WSDL fetching during the rollout period.

Reply:
yeah, I definitely want to put this on a flipper feature for a safe introduction / testing phase. On a sync now with @acovrig and discussing. - thanks for the heads up and good eye @nihil2501